exasol/ds/sandbox/runtime/ansible/roles/jupyter/files/notebook_requirements.txt

```diff
@@ -9,5 +9,5 @@ wordcloud==1.9.4 # required for Text AI notebooks
 pickleshare==0.7.5 # See https://github.com/exasol/ai-lab/issues/291 for details.
 ipyfilechooser==0.6.0 # required for SLC notebooks
 ipywidgets==8.1.5 # enable interactive Javascript widgets in the notebooks
-exasol-notebook-connector==2.0.0
+exasol-notebook-connector [sqlalchemy, pyexasol, bucketfs, docker-db, slc, ibis, transformers, text-ai, sagemaker] ==2.2.0
```

Comment:
Updated dependency to exasol-notebook-connector (NC).

Since version 2.1.0 most transitive dependencies of NC are optional (pip "extras") and need to be specified explicitly for installation.